README.md

```diff
@@ -23,12 +23,19 @@ initialized from there. The simulation can be viewed from different positions
 by dragging the mouse and using the mouse wheel to control the camera.
 
 ### Fluid Simulation
-This demo visualizes fluid behaviour in a closed container. Each cell in the
+This demo visualizes fluid behavior in a closed container. Each cell in the
 cellular automata represents a fluid particle existing in a velocity field.
-Drag mouse around the screen to create fluid particles with velocities in
-direction of mouse travel. Fluid fades slowly over time so as not to fill
+Drag the mouse around the screen to create fluid particles with velocities in
+direction of the mouse travel. The fluid fades slowly over time so as not to fill
 the container.
 
+### MPI for CUDA Backend 
+The source code within provides two SYCL example programs; send_recv_buff.cpp, send_recv_usm.cpp, which are CUDA MPI aware. One example uses the SYCL Unified Share Memory (USM). The other uses the Buffer (buff) model. Each example uses the programing pattern Send-Receive. 
```

Comment:
```suggestion
The source code within provides two SYCL example programs; send_recv_buff.cpp, send_recv_usm.cpp, which use CUDA-Aware MPI. One example uses the SYCL Unified Share Memory (USM). The other uses the Buffer (buff) model. Each example uses the programing pattern Send-Receive. 
```

nit: personally I prefer to not expand USM to Unified Shared Memory because it would make me think it is only for malloc_shared: i.e I think avoiding its terrible naming being made quite so obvious might make it a bit less confusing.

Reply:
Fix reword 'use CUDA-Aware'.
Done.